### ISSUE TITLE
Change the width and height during streaming 

### ISSUE THREAD

**Issue by EbramSamy** [radioman/WebRtc.NET#26]
After starting streaming and setting SetVideoCapturer(width,height) I want to change the height and width before pushing new frame during streaming ?!

**Comment by radioman** [comment]
does webrtc support that?

**Comment by EbramSamy** [comment]
yes I think because you can support MaxWidth and MaxHeihght as a video constraints 

http://stackoverflow.com/questions/26415778/getusermedia-video-size-in-firefox-chrome-differs

**Comment by Zydraonex** [comment]
As I know the client first sends his recommendet video constraints and then you can add a matching track to the Stream. It's not possible to change a running track you have to add a new track and maybe delete the old one. At the clients side you can change the displaying track easily with JS on the fly, so the end user doesn't get notice about it.
